app/messages/Message.scala

```diff
@@ -18,7 +18,7 @@ package messages
 
 import langswitch.{Language, Languages}
 
-final case class Message(
+final case class Message private (
```

Comment:
this may not do what you think it does - I think you'd still be able to do this:
```scala
Message("english", Some("welsh"))
```
but you're just prevented from doing this:
```scala
new Message("english", Some("welsh"))
```
I'll have a double check...

Reply:
yeah, I think that's the case after a little test locally

Reply:
Yup, thanks for pointing this out. I think I've fixed that.